templates/zerver/integrations/github-actions.md

```diff
@@ -1,8 +1,21 @@
-It's easy to send Zulip messages from GitHub Actions! This is useful:
+# Zulip GitHub Actions integration
 
-  * If you want to report the success or failure of an Action to Zulip.
-  * If you want to send output from your actions to Zulip.
+It's easy to send Zulip messages from GitHub Actions!
 
-Instructions are available [here](https://github.com/zulip/github-actions-zulip#readme).
+{start_tabs}
 
-See also the [GitHub integration](/integrations/doc/github).
+This is useful:
+
+  1. If you want to report the success or failure of an Action to Zulip.
+
+  1. If you want to send output from your actions to Zulip.
+
+Instructions are available [here][instructions].
```

Comment:
Though we could move those instructions here, I thought the tabular format used there was neat, so I didn't.